.github/workflows/codeql-daily.yml

```diff
@@ -2,9 +2,16 @@ on:
   schedule:
   - cron: '0 12 * * 4'
 
+permissions:
+  contents: read
+
 jobs:
   CodeQL-Build:
 
+    permissions:
+      actions: read  # for github/codeql-action/init to get workflow details
+      contents: read  # for actions/checkout to fetch code
```

Comment:
i dont think these 2 are necessary - checking the codeql readme it suggests these are only required for private repos

Reply:
Done.